opencti-platform/opencti-front/script/extract-label-keyword.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const util = require('util');
+const path = require('path');
+
+const readdir = util.promisify(fs.readdir);
+const stat = util.promisify(fs.stat);
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+
+const srcDirectory = '../opencti-graphql/src/modules';
```

Comment:
This will only concerns modules then ? "old" files are not parsed

Reply:
if it's the only one, we could just inject it during parsing explicitely...

Reply:
done